containers/podman-desktop
# fix: set correct pod deletion status

### What does this PR do?

Kubernetes pods never go into a deletion/termination status - they stay in 'running' state while the containers are given time to shut down, then the pod just disappears. Kubectl uses the deletion timestamp to show these pod as Terminating:
https://github.com/kubernetes/kubernetes/blob/a9f6b93b62a98598913180c640ab044c85a6718b/pkg/printers/internalversion/printers.go#L779C39-L779C39

This change uses the same mechanism as kubectl to detect this condition, but uses our existing DELETING state. Also removed a duplicate comment I noticed.

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

Partial fix for issue #3529, the rest is showing deleting state in the UI.

### How to test this PR?

Delete a pod running on Kubernetes; you'll need to use the tooltip on the status icon to see the DELETING state.
